src/olympia/files/tests/test_models.py

```diff
@@ -1065,6 +1065,36 @@ def test_webextension_crx_large(self):
         assert upload.path.endswith('.xpi')
         storage.delete(upload.path)
 
+    def test_webextension_crx_version_3(self):
+        """Test to ensure we accept CRX uploads (version 3), but convert them
+        into XPI files ASAP to keep things simple.
+        """
+        upload = self.get_upload('webextension_crx3.crx')
```

Comment:
Is it practical to test this in the same way as the following tests, with `FileUpload.from_post`?

Reply:
It's possible but I wanted to follow what the original tests for successful scenarios did